test/export/test_export.py

```diff
@@ -4997,42 +5003,97 @@ def forward(self, x, y, z):
             torch.randn(48),
         )
         dynamic_shapes = {
-            "x": [Dim(f"dx{i}") for i in range(2)],
-            "y": [Dim(f"dy{i}") for i in range(2)],
-            "z": [Dim(f"dz{i}") for i in range(1)],
+            "x": [Dim(f"dx{i}", min=2) for i in range(2)],
```

Comment:
what happens without the `min=2`?

Reply:
There's just an annoying suggested fix thing which will suggest refinement with min = 2 if unspecified (since analysis is done with min = 2). Putting this in so it matches the fix, and the fix doesn't show up. But this should be fixed in the automatic dynamic shapes PR.

The bigger concern I think is with having min=4 for dz - this has to do with the min = 0 or 1 thing. If we get a guard s0 * s1 = s2, since we do analysis with min=2, s2 has a refined min of 4. 

I think I'll revisit how we handle min = 0/1, now that I understand symbolic_shapes better.